### ISSUE TITLE
NVDA Always Announces entire Line Using IME in Windows 11 Notepad

### ISSUE THREAD

**Issue by dangero2000** [nvaccess/nvda#17319]
### Steps to reproduce:
* Open Notepad on Windows 11
* Enable IME input, (Microsoft and/or Google IME doesn't matter, only tested with Japanese IME)
* Type a character or full word/phrase and press enter, (selecting a candidate with the arrow keys is optional)
* Type some more text and press enter
### Actual behavior:

I typed to (と) in Hirigana, then pressed enter. NVDA read to (と). I then wrote the same thing again and pressed enter. NVDA read toto (とと).
### Expected behavior:

NVDA should have read to (と) and not the entire line.
#### NVDA installed/portable/running from source:
Any installation method will produce this bug.
#### NVDA version:
2024.3.1
#### Windows version:
Windows 11 24H2
#### Name and version of other software in use when reproducing the issue:
Microsoft.WindowsNotepad version 11.2408.12.0
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have tried 2024.1 and 2019.3.1 and received the same results. The NVDA version does not effect the results in anyway; only the Notepad version.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by XLTechie** [comment]
CC @josephsl

**Comment by SaschaCowley** [comment]
Is this possibly related to #17253?

**Comment by cary-rowen** [comment]
I think this may be the same issue as #17253

**Comment by dangero2000** [comment]
I'm not sure. I don't believe so. This bug only takes place in Notepad, and it reports the entire line, not necessarily the name of the edit box.
